base/applications/rapps/gui.cpp

```diff
@@ -304,9 +569,17 @@ class CAppInfoDisplay :
         return FALSE;
     }
 
+    VOID ResizeChildren(int Width, int Height)
+    {
+        int SnpshtWidth = SnpshtPrev->GetRequestedWidth(Height);
+        ::MoveWindow(SnpshtPrev->m_hWnd, 0, 0, SnpshtWidth, Height, TRUE);
+        ::MoveWindow(RichEdit->m_hWnd, SnpshtWidth, 0, Width - SnpshtWidth, Height, TRUE);
```

Comment:
When resizing multiple child windows, it's better to use `BeginDeferWindowPos`, `DeferWindowPos` and `EndDeferWindowPos`.
This will move all windows at once.
See for an example MainWnd_OnSize from fontview:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/fontview/fontview.c;hb=5d88487cd18058baad1199277777e148c8a1081a#l402

Reply:
ok, now I improved this.